scripts/run_pretty_printer_tests.py

```
@@ -0,0 +1,121 @@
+        return Arhitecture.ARM64
+    else:
+        return Arhitecture.Other
+
+
+def set_executable(filepath: str):
+    s = os.stat(filepath)
+    # TODO: it seems `os.chmod` doesn't work on Windows with executable permissions
+    os.chmod(filepath, s.st_mode | stat.S_IEXEC)
+
+
+def run_lldb_tests(clion_version: str):
+    host_os = current_os()
+    arch_name = current_arhitecture().value
+
+    if host_os == OS.MacOs:
+        # TODO: Remove after CLion snapshot builds provide these files with required permissions
+        lldb_mac_bin_dir = path.abspath(f"deps/clion-{clion_version}/bin/lldb/mac")
+        set_executable(f"{lldb_mac_bin_dir}/lldb")
+        set_executable(f"{lldb_mac_bin_dir}/LLDBFrontend")
+        set_executable(f"{lldb_mac_bin_dir}/LLDB.framework/LLDB")
+
+        # TODO: Use `lldb` Python module from CLion distribution
+        lldb_path = "/Applications/Xcode.app/Contents/SharedFrameworks/LLDB.framework/Resources/Python"
+        python = "python"
+    elif host_os == OS.Linux:
+        lldb_path = path.abspath(f"deps/clion-{clion_version}/bin/lldb/linux/{arch_name}/lib/python3.8/site-packages")
+        # BACKCOMPAT: 2022.3. Since 2022.3.2 `os.path.exists(lldb_path)` should be always true
+        if not path.exists(lldb_path):
+            lldb_path = path.abspath(f"deps/clion-{clion_version}/bin/lldb/linux/lib/python3.8/site-packages")
+        python = "python3"
+    elif host_os == OS.Windows:
+        lldb_bundle_path = path.abspath(f"deps/clion-{clion_version}/bin/lldb/win/{arch_name}")
+        # Create symlink to allow `lldb` Python module perform `import _lldb` inside
+        # TODO: Drop when this is implemented on CLion side
+        os.symlink(f"{lldb_bundle_path}/bin/liblldb.dll", f"{lldb_bundle_path}/lib/site-packages/lldb/_lldb.pyd")
+
+        lldb_path = f"{lldb_bundle_path}/lib/site-packages"
+        python = f"{lldb_bundle_path}/bin/python.exe"
+    else:
+        raise Exception("Unsupported OS")
+
```

We could check that `lldb_path` exists and fail fast if not (same with `gdb_binary` in `run_gdb_tests`)